buildSrc/src/main/kotlin/SqlDelightTmdbCacheDefinitionsGenerator.kt

```diff
@@ -0,0 +1,110 @@
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+
+/**
+ * A Gradle Task to generate the SQLDelight definitions related to the Tmdb Cache.
+ */
+open class SqlDelightTmdbCacheDefinitionsGenerator : DefaultTask() {
+    /** Where to generate the .sq files */
+    private val sqldelightRoot = File(project.projectDir, "src/androidMain/sqldelight/tmdbCache/io/github/couchtracker/db/tmdbCache")
+
+    /** Common columns */
+    private val movieIdColumn = SqlColumn.int("tmdbId", "io.github.couchtracker.tmdb.TmdbMovieId")
+    private val languageColumn = SqlColumn.text("language", "io.github.couchtracker.tmdb.TmdbLanguage")
+
+    /** Which caches to create */
+    private val caches = listOf(
+        SqlTable(
+            name = "MovieDetailsCache",
+            key = listOf(movieIdColumn, languageColumn),
+            value = SqlColumn.text("details", "app.moviebase.tmdb.model.TmdbMovieDetail"),
+        ),
+        SqlTable(
+            name = "MovieReleaseDatesCache",
+            key = listOf(movieIdColumn),
+            value = SqlColumn.text("releaseDates", "kotlin.collections.List<app.moviebase.tmdb.model.TmdbReleaseDates>"),
+        ),
+    )
+
+    /** Which sql functions to generate */
+    private val sqlFunctions = listOf(
+        SqlFunction("get") { cache ->
+            append("SELECT ${cache.value.name}\n")
```

Comment:
Can we use [appendLine](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/append-line.html) here? It avoids having to remember the ugly \n at the end of each line